changelog/11680.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
So Chelsea and I talked about doing one large status update.  Maybe just change this to something more generic that we can use for all the partial updates.  e.g. update partials to components.